src/app/contribution-graph/contribution-graph.component.ts

```diff
@@ -1,4 +1,5 @@
 import { Component, Input, OnInit } from '@angular/core';
+import { deflateSync } from 'zlib';
```

Comment:
What's this for? Please remove if not used in the code.

Reply:
I didn't add it myself, so I don't know what it does. Must have be automatically added. should I still remove it ?

Reply:
Yes, please remove. Might have been added by visual studio code by mistake.